### DISCUSSION TITLE
1.0: App Store Release

### DISCUSSION THREAD

**Discussion by EthanLipnik** [EthanLipnik/OpenSesame#30] [Announcements]
OpenSesame has been released to the App Store!

Features
- iCloud Syncing
- OTP Auth (Two Factor Authentication)
- Safari Autofill
- On-Device AES encryption
- Biometrics unlock
- Pinned accounts
- Multiple vaults
- Multipeer account sharing
- Import/Export
- Password generator
- Credit/Debit card support
- Notes field
- OTP QR codes
- Alternate icons

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenSesameManager/OpenSesame/releases/tag/1.0'>1.0: App Store Release</a>.</em>

**Comment by ihostage** [comment]
Hi, Ethan! 👋 
Thank you for your work 👍 
When do you plan a publish the Mac OS version of Open Sesame to the Mac App Store?

**Reply by EthanLipnik** [reply]
Hi, yes I do! I just need to finish the App Store marketing for it
